### PR TITLE
git 2.4.0

### PR DESCRIPTION
**Pull request by ianferguson**

### CODE REVIEW

File: Library/Formula/git.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Git < Formula
   homepage "http://git-scm.com"
```
Comment (DomT4) [inline Library/Formula/git.rb]: Can you poke this to ` https:// `? Thanks!

Comment (ianferguson) [inline Library/Formula/git.rb]: @DomT4 seems like that should be done as a separate commit/change/PR from just bumping the versions doesn't it? 

That being said, I'm happy to tack on the change to this

Comment (ianferguson) [inline Library/Formula/git.rb]: @DomT4 done.

Comment (DomT4) [inline Library/Formula/git.rb]: Na. Opportunistic changes are fine in PRs, especially plaintext => SSL/TLS link upgrades.

Comment (ianferguson) [inline Library/Formula/git.rb]: Cool, I'll keep that in mind for the future. I also just reread contributing.md and as a result squashed that change into one commit. let me know if there's anything else you spot.

Comment (DomT4) [inline Library/Formula/git.rb]: Nice! Looks great :tada:.

**Comment by joshfriend** [comment]
:+1:
